apps/ledger-live-mobile/fastlane/Fastfile

```diff
@@ -130,6 +130,18 @@ platform :ios do
       keychain_password: ENV["CI_KEYCHAIN_PASSWORD"],
       git_basic_authorization: Base64.strict_encode64("#{ENV["GIT_REPO_USER"]}:#{ENV["GH_TOKEN"]}"),
     )
+    match(
+      type: "appstore",
+      app_identifier: ["com.ledger.live.nightly"],
```

Comment:
is it possible to make it like this 
`app_identifier: [ENV["APP_IDENTIFIER"]]`?

Reply:
this is a separate lane for syncing the profiles. We do want to hardcode this for every profile we want to sync, so we get the full range of profiles synced. (If we set it as `ENV["APP_IDENTIFIER"]` this would only retrieve 1 profile, the one for the current env at runtime of this code)

Reply:
ah ok, understood, so while it's a single app_identifier we can simply remove the array and do it like this `app_identifier: "com.ledger.live.nightly",` can we ?

Reply:
Yep, good point! I'll update